.travis.yml

```diff
@@ -40,3 +42,5 @@ env:
   - secure: "gcQRNT0MsN/GRF71rmO6ElUu2w8ZV3erHC0G982vSiT6bQqN4f3H80D+lHH5NkCqjMKjTNksjl06g8ZzlxZUSslAh7QuVqwp7oTTnrddG3wmWGmfXBnw55Zsfs1uLdlWvhRWr4tCJ04HkojL+J5XV9T1T7U97VB7EONcYVtDogLO4lE7UUVenT963ULp8B9jJkpSqPDow7oxb//b4rElPf99zA5cfzgWGDfUHkkVTc5B20SyPgbuQ181t0MIj9zaBAkaYVWk/2nkNIC7Yf/rlhN6lpRqE9/eB7+jl2J7Kn46KqioaS/i8FOZz7iMDFTx2Pl+Pz40i0MYgc+nGFN3JhRtEGcaLCgrEoSc5mhsKCIZpEWm02oLsBZ+zSCuVkgPBMB2m/ciWppqkPdjLD8Dz890H1OVvIu912BAVknvcJPCF8XtKoV/fkxJ3eOGKMy9medbo7up141HtkPIwKkLmi3B5U5TSpSExUPSBf3v/1Hu/6XdtZxps5YmOxn7T8HzC0fompjK2GJ0YUI7ZcEd8+EPi4+AogZ3OFVtJ68q5kqXnbs08VIBhhrGZUCM3kWFKyIyrESI1QjWk2l/WWBxyMxMynP2d1wH8dedsr9BYePBU8AdmWYLLrW/DeH7rkHUzIit+OjL1M70TsvKV0rym1iuTuA6+DZfn8FUvNwxyZI="
   - secure: "Gw781tVA6gPXVyho7bNCdAe/knVBUdwl8DTcpYIpRnXfKhccIu19N7v7diA6LtZtbb1qxyDGREshF8OBHwriHm+raGIwWkVwVoAuotAHoxe4jC5VRy+QbWznUPOOtdSYTpoBVkPrHJcNTdA2GjKnjwNgsEbCUDSpneeI8vvPCT248sRLH6ncev0sk3yhhk1dH6mpiwfxS5mCeF3LeQ3fUMsnrEu6IN9oAE1nuOgu17igSFajZ7/oJTLPajFjCUY868Ro0rink94yVGUoGRfxCwipLiI0ihBaekFhXF3zQh8D4IAJWZOt/rKT7f1hGecDTbqMwS/WTFoirVU20CxKJ7b5V6JLtmD8QTTjohzUhYEnAdyaZBCDFEfKPZi88XO9XvDxufaFEccXtOzE6egEbmaVlJSFFqbuS6/GTGty7q/mu1L0ZKSlCxHy6UB5xRHbFjzJmQXahYiYI78weXOzxNkoqc4LomlOK4unV7LIvzBPyr71L+1IkKR09mS7FHV9tXkqOz2+j6+b8AEXs7zb1IKxfciYWDVNSKJ7ABA9kTXokC1MoLIHRmyNNefHtbCCPfSKHE3qDF4T0fJgX2KRc/wrYf4y0x+dYgeKBPy6RQN5ql2y34N/05DttinbwEQEZ1u7FOKzBD7lKZtB15IlUp6CRCat+tnz+c/8RfGmS+c="
   - secure: "wTy5E4DD83S5aRgaXjgiZvbstMJK736h2FVVRA/EW0jsc/BLwYagSWgh84B+MKub10/JEvlLizvCOY81JiV8uf9IKxGnxaj3Lal5cqxF3USzvyDsdJsxI7Ek87PcN/34LQyC/nzk7kEC00ibxllvEQRJHyNEOsnuM5gQTQPb6HftdMP6wMwBziqvMwhgOORN4S1nwwlO4a22A0ouV98dzIi627ehQs9+gRK99bzjD1n7GjGrh6Fkkb593MQxNBLjnTgy2Ots2Iootu1j6KtOF+adyNugyXL9Yfbmg0MXkWJzbaDc8ZAX8Hmv3FzjIKPUmLyLJU5JcxQ8Ivmu45MHGnOOhV/KSGdU/xcT15UoaoRAbUVUBGVljWR9cFVFZHbAVDuWFEyFAbW1gEaqPzfoTsJs1wjL5cnQ2MHhm8KqGqCCwV2RjK+4Yi73rNmh8zrSaFEEgrPzXqrvhwio1dG27AmyCoDWum7mQtDSvqyW+POhG88Y+5aIh49EMyar/4DndX6PFa+Ot0oIrwC50DOEzSOYSt5+KfOjl0yAC1p3KsKgpVPZs6VKHel97ZBYXVTLpbr56e/jBam9fHqeAHkdcqVQsmM9mXTbV7ixo0dazf6q7fIYYy6/beIfep73jy8k6Ra8SmSYwQJsvAKC8MUqlSkO0C5qFdMjDrK4DmM1AoE="
+  - secure: "uawiw2n/e4piC64CShMbLk9nHOpb7J5FDBmH0pqVDb/aF6DqCV/ceo9ODKux58Qi5vRQFiKGmgneycWIT6SHlU4Jcs3EY5ex2mqMaIXUgHRHKqPbBnh5ORPUCfvyF3K/mGUi2pIcoIFKra/zKQv58+90nMg2Vs3d9yunP1Q44K8DZUzwUNR3e3sElFfLXw7KIR0DZX7/Ybw5v6v7+l3lmI72AFGFIkL3wp7GjdCOx0hVNO1ycE12WcXpzIAYLKqPpjSW6ty/ML8ahdFh49dGY9cCfeOdbevarTUS/zO0Jad8vbl8NMIyyb2m+wWOTrgXte87aLfGypPIVmbtbR4eS/9gH0bY8YiFcl9i9wBsXdAHridWQAnBRN5M4GcjtU0nSY5esnd8Lfu/x3WcyVwR+8GUsGby+dpbBUwxdhq/PaTK/ws4eoVQJmyNF0ifKBD3UkcqlD4SHZsn64XplMxrO2NlDPoDy1cpLfBntwpD926c2bJX9m0j8v29LnM/2WxyOF8NBKVgvOcQn4l1ema6STl8sy0W+o+WEWPtLQlFHKY86DzaG/vFUTSFhLtKB71vh6mhRsIBxvfnG+69+2FCfcu4ii2SZHb3dhWwZFBqwIdRSiCX8nU6xQ7WJbT57XXjCBRJQWUDmL+uy9YsIcB4OEdk+0j45Ang7d5NlawfD+s="
```

Comment:
This is setting LOGGLY_TOKEN right? You've forgotten to actually include that in the `terraform apply` command on line 23.

Alternatively you could set an env var `TF_VAR_loggly_token`. But it would be odd to do that for just loggly_token. So either do that for all the vars, or explicitly pass them all on on line 23.